service/server.py

```diff
@@ -84,7 +84,3 @@ def get_properties(postcode):
         return jsonify(result)
     else:
         return jsonify({'titles': []})
-
-if __name__ == '__main__':
-    port = int(os.environ.get('PORT', 5000))
```

Comment:
Looks like we never run the application this way. In dev mode it's started by `run-dev.py` and elsewhere it's done by gunicorn.